lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/statemachine/NestedJourneyTypes.java

```diff
@@ -13,7 +13,8 @@ public enum NestedJourneyTypes {
     ADDRESS_AND_FRAUD("address-and-fraud"),
     KBVS("kbvs"),
     STRATEGIC_APP_TRIAGE("strategic-app-triage"),
-    WEB_DL_OR_PASSPORT("web-dl-or-passport");
+    WEB_DL_OR_PASSPORT("web-dl-or-passport"),
+    DCMAW_DRIVING_LICENCE("dcmaw-driving-licence");
```

Comment:
Looks like the comment above has failed 😱 - we need to update the variable in the journey map visualisation as well!

Reply:
and I wrote this too 🤦‍♀️ I think it may have just been me not reading the code properly